src/Spectre.Algorithms/Results/DivikResult.cs

```diff
@@ -27,17 +28,197 @@ namespace Spectre.Algorithms.Results
 	/// </summary>
 	public sealed class DivikResult
     {
-	    private object _matlabResultStruct;
+        #region Properties
+        /// <summary>
+        /// clustering quality index
```

Comment:
Please start documentation summaries with an uppercase letter. (Here and in other properties below)